lib/grpc/GrpcService.ts

```diff
@@ -499,9 +499,17 @@ class GrpcService {
    */
   public listCurrencies: grpc.handleUnaryCall<xudrpc.ListCurrenciesRequest, xudrpc.ListCurrenciesResponse> = (_, callback) => {
     try {
-      const listCurrenciesResponse = this.service.listCurrencies();
+      const currencies = this.service.listCurrencies();
       const response = new xudrpc.ListCurrenciesResponse();
-      response.setCurrenciesList(listCurrenciesResponse);
+
+      currencies.currencies.forEach((currency: CurrencyInstance) => {
+        const resultCurrency = new xudrpc.Currency();
+        resultCurrency.setDecimalPlaces(currency.decimalPlaces);
+        resultCurrency.setCurrency(currency.id);
+        resultCurrency.setTokenAddress(currencies.tokenIdentifiers[currency.id] || '');
+        resultCurrency.setSwapClient(currency.swapClient === SwapClientType.Lnd ? xudrpc.Currency.SwapClient.LND : xudrpc.Currency.SwapClient.RAIDEN);
```

Comment:
You can just do currency.swapClient as number here - our internal enum matches the enum defined in the rpc proto.